x-pack/filebeat/module/cisco/ios/pipeline_test.go

```diff
@@ -2,6 +2,8 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+//go:build !requirefips
```

Comment:
Current failure indicates:
```
pipeline_test.go:186: failed in processor.javascript: TypeError: Object has no member 'CommunityID' at config/pipeline.js:158:21(188)
```

@kruskall, you disabled the community_id functionality for FIPS mode, do you know what could be causing this failure?

Reply:
For some reason I thought the community_id processor had been rendered into a no-op under fips, but actually it won't be present -- https://github.com/elastic/beats/blob/07103508dcbba7d8d435b220a09a71fa36aa1be9/libbeat/processors/communityid/communityid.go#L18

So that error makes sense in that it cannot call the processor. But I'm not sure how this should be handled. It's more of a requirements question, should the cisco module continue working in FIPS mode (just without the community_id capability) or should it be disabled (and probably not included in the FIPS distribution artifact).

Reply:
Does that processor have to be diabled? the flowhasher it uses should return an empty string:
https://github.com/elastic/beats/blob/d274aa9aedbc0edc8ec53bb484689d5ca6ac67cd/libbeat/common/flowhash/communityid_fips.go#L32-L35